lambda_2/lambda_function.py

```diff
@@ -34,8 +34,54 @@ def process_json(json_content):
     flattened_records = []
 
     for item in json_content['list']:
+        location = item['location']
+        latitude = None
+        longitude = None
+        country = None
+
+        if location == 'London':
+            latitude = 51.5098
+            longitude = -0.1180
+        elif location == 'Paris':
+            latitude = 48.8588897
+            longitude = 2.3200410217200766
+        elif location == 'Brussels':
+            latitude = 50.8465573
+            longitude = 4.351697
+        elif location == 'Madrid':
+            latitude = 40.4167047
+            longitude = -3.7035825
+        elif location == 'Budapest':
+            latitude = 47.48138955
+            longitude = 19.14609412691246
+        elif location == 'Oslo':
+            latitude = 59.97239745
+            longitude = 10.775729194051895
+        else:
+            latitude = 0.0
+            longitude = 0.0
+
+        if location == 'London':
+            country = 'United Kingdom'
+        elif location == 'Paris':
+            country = 'France'
+        elif location == 'Brussels':
+            country = 'Belgium'
+        elif location == 'Madrid':
+            country = 'Spain'
+        elif location == 'Budapest':
+            country = 'Hungary'
+        elif location == 'Oslo':
+            country = 'Norway'
+        else:
+            country = 'N/A'
```

Comment:
I think a cleaner way to do this would be with a dictionary lookup. 
e.g. `LOCATION_LOOKUP = {"London": {"Country": "United Kingdom", "Long": -0.1..., "Lat"...},...`,
then you just extract the vals e.g. `country = LOCATION_LOOKUP[location]["Country"]` and `longitude = LOCATION_LOOKUP[location]["Longitude"]` ... etc